aws/arctic-wolf/arctic_wolf.tf

```diff
@@ -0,0 +1,212 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "s3_bucket_versioning" {
+  description = "S3 bucket versioning enabled?"
+  default     = false
+}
+
+variable "sse_algorithm" {
+  type        = string
+  default     = "AES256"
+  description = "The server-side encryption algorithm to use. Valid values are `AES256` and `aws:kms`"
+}
+
+variable "kms_master_key_arn" {
+  type        = string
+  default     = ""
+  description = "The AWS KMS master key ARN used for the `SSE-KMS` encryption. This can only be used when you set the value of `sse_algorithm` as `aws:kms`. The default aws/s3 AWS KMS master key is used if this element is absent while the `sse_algorithm` is `aws:kms`"
+}
+
+variable "enable_guardduty_logging" {
+  type        = bool
+  default     = false
+  description = ""
+}
+
+variable "vpc_id" {
+  type        = string
+  default     = null
+  description = ""
+}
+
+variable "guardduty_detector_id" {
+  type        = string
+  default     = null
+  description = ""
+}
+
+variable "flow_log_filter_pattern" {
+  type        = string
+  default     = null
+  description = ""
+}
+
+variable "vpc_flow_log_group_name" {
+  type        = string
+  default     = null
+  description = ""
+}
+
+variable "cloudtrail" {
```

Comment:
Is this a name, id, arn? Add a suffix to make the var more descriptive.

Reply:
changed to cloudtrail_name and all subsequent references